go.sum

```diff
@@ -0,0 +1,17 @@
+github.com/ajstarks/svgo v0.0.0-20180226025133-644b8db467af/go.mod h1:K08gAheRH3/J6wwsYMMT4xOr94bZjxIelGM0+d/wbFw=
```

Comment:
Just a question: how this file was generated (it doesn't look like it was typed by hand)? Thanks!